h/views/activity.py

```diff
@@ -92,31 +95,76 @@ def group_search(request):
 
     return result
 
+
+@view_config(route_name='activity.user_search',
+             request_method='GET',
+             renderer='h:templates/activity/search.html.jinja2')
+def user_search(request):
+    if not request.feature('search_page'):
+        raise httpexceptions.HTTPNotFound()
+
+    username = request.matchdict['username']
+
+    result = search(request)
+
+    result['opts'] = {'search_username': username}
+    result['more_info'] = 'more_info' in request.params
+
+    user = request.find_service(name='user').fetch(
+        'acct:{username}@{authority}'.format(username=username,
+                                             authority=request.auth_domain))
```

Comment:
I've done this now but suggest that I push it as a separate PR once this one is merged